packages/rainbow-button/src/components/RainbowButton.tsx

```diff
@@ -10,8 +11,8 @@ export function RainbowButtonProvider({
   'chains' | 'avatar' | 'initialChain' | 'modalSize' | 'showRecentTransactions'
 >) {
   return (
-    // chains array is intentionally left empty
-    <RainbowKitProvider chains={[]} {...options}>
+    // Wagmi requires 'chains' as a tuple type with at least one chain
+    <RainbowKitProvider chains={[mainnet]} {...options}>
```

Comment:
If `chains` in `RainbowKitProvider` is removed or optional, we can drop this need